compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -124,6 +124,13 @@ void sym_wquant_per_channel(CircleConst *node, std::vector<float> &min, std::vec
 
   auto quantize = [&](uint32_t *indices, loco::TensorShape &dimension, int channel_dim_index) {
     int channel_idx = indices[channel_dim_index];
+
+    // Small scale factor produced by small weights, which can be quantized to 0 with scale 1.0f
+    if (scaling_factor[channel_idx] < std::numeric_limits<float>::epsilon())
```

Comment:
I'm not sure it's best condition for rounding to 0. May be better to use value lower than `epsilon()`?

Reply:
I added TODO comment. I think better to find better design latter!